docs/docs/reference/other-new-features/trait-parameters.md

```diff
@@ -56,11 +59,13 @@ class E extends Greeting("Bob"), FormalGreeting
 ### Traits With Context Parameters
 
 This "explicit extension required" rule is relaxed if the missing trait contains only
-[context parameters](../contextual/using-clauses). In that case the trait reference is
+[context parameters](../contextual/using-clauses.html). In that case the trait reference is
```

Comment:
The corresponding file in the docs.scala-lang.org repo uses a different type of link. See https://github.com/scala/docs.scala-lang/blame/main/_scala3-reference/other-new-features/trait-parameters.md#L61

Would that type of link be supported? IIRC the difference is that those links are checked by Jekyll (so developers know early if they wrote an incorrect URI) instead of being only checked in the CI by htmlproofer.

Reply:
Do you mean the notation with `[lable]({% link ... %})`? I replaced all suck links because they caused liquid templates to fail. I should create a custom handler for such links to not preprocess them, just leave them raw.

Reply:
Yes, those links.

> they caused liquid templates to fail

We need some type of support because that’s the best way to create links in Jekyll. Note that we don’t need to properly render them since anyway scaladoc is not going to render these pages.

Reply:
I believe overriding links handler with some guards checking for this syntax `{% %}` should not be a problem, but I have to double check it